portfolio/src/main/webapp/script.js

```diff
@@ -92,25 +92,34 @@ function getGuestBook() {
   const guestBook = document.getElementById('guest-book-comments');
   fetch("/data").then(response => response.json()).then((comments) => {
     comments.forEach((comment) => {
-      const commentHolder = document.createElement('div');
-      const commentText = document.createElement('p');
-      const commentAuthor = document.createElement('p');
-      
-      commentHolder.className = "comment";
-      commentText.className = "message";
-      commentAuthor.className = "author";
-
-      commentText.innerText = comment.text; 
-      commentAuthor.innerText = comment.author;
-
-      commentHolder.appendChild(commentText);
-      commentHolder.appendChild(commentAuthor);
-      guestBook.appendChild(commentHolder);
-      guestBook.appendChild(document.createElement('br'));
+      guestBook.appendChild(createComment(comment));
+      guestBook.appendChild(document.createElement('br')); 
+        /* Add break before next comment - one extra at the end is actually
+        good, makes it easier to scroll down to read the last comment */
     });
   })
 }
 
+const months = ['January', 'February', 'March', 'April', 'May', 'June', 
```

Comment:
You could use something like [SimpleDateFormat](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html) to format the dates instead of converting it yourself.

Reply:
Oh, that's a great idea, thanks!